src/main/resources/application.yml

```diff
@@ -26,6 +26,10 @@ management:
   info:
     defaults:
       enabled: false
+  endpoints:
+    web:
+      exposure:
+        include: health, info
```

Comment:
Was this changed in a recent spring-boot release? Previously enabled by default, now disabled?

Reply:
Yes, in 2.5 release for security reasons: https://github.com/spring-projects/spring-boot/wiki/Spring-Boot-2.5-Release-Notes#secure-info-endpoint

In terms of security. we hide the default values from Actuator and insert our own.

Reply:
Understood. Thanks for the clarification.